cyberark/conjur
# 693 reduce iam logging

#### What does this PR do?
Removes detailed logging of IAM authentication

#### Any background context you want to provide?
See ticket

#### What ticket does this PR close?
Connected to https://github.com/conjurinc/appliance/issues/693

#### Where should the reviewer start?
[Jenkins Build](https://jenkins.conjur.net/job/cyberark--conjur/job/693-reduce-iam-logging/)

#### How should this be manually tested?
- Create appliance VM
- Create VM client with IAM role attached
- Add [policies](https://conjurhq.slack.com/archives/CA8AT283D/p1550263580090800) for IAM authentication to appliance
- `docker logs -f conjur-appliance` in the appliance
- Run [`test.sh`](https://gist.github.com/sgnn7/f3156d750ce8dd89e82bd4ae4a1feed2)
- Ensure that appliance doesn't show anything related to token params during authentication

#### Link to build in Jenkins (if appropriate)
Connected to https://github.com/conjurinc/appliance/issues/693

#### Has the Version and Changelog been updated?
Yes

#### Questions:
> Does this work have automated integration and unit tests?
No - related issue [here](https://github.com/conjurinc/appliance/issues/696)

> Can we make a blog post, video, or animated GIF of this?
No

> Has this change been documented (Readme, docs, etc.)?
Yes

> Does the knowledge base need an update?
